phpstan.dist.neon

```diff
@@ -57,6 +57,6 @@ parameters:
         -
             identifier: offsetAccess.notFound
             path: .
-            count: 541
+            count: 542
```

Comment:
FYI #8103 should fix this and some more such errors

Reply:
In #8107 I am going to change approach to `offsetAccess.notFound`, because as you already experienced it's not friendly because when you hit it, you don't know where the problem is.

Reply:
Thank you - yes, finding the phpstan error is/was otherwise not easy.